scripts/setup_ubuntu.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
```

Comment:
This is totally up to you (and is something that would need to be changed for Mac's `setup.sh` as well, but don't worry about that)---because we're bypassing explicit user approval for each `sudo apt-get install` using `-y`, we should probably tell the user what it is we're installing on their computer. Maybe include something like this right after this line?
```bash
echo "This script will install the following packages on your machine:"
echo "mongodb, libcairo2-dec, yarn, nodejs"
read -p "Is this okay (y/n)? " -r # Read in user input
if [[ ! $REPLY =~ ^[Yy] ]] ; then # If the input doesn't begin with 'y' or 'Y', exit with code 1
  echo "Stopping setup."
  [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
fi
```

Reply:
This didn't seem to work, and so was not used.